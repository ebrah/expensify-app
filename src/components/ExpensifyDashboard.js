import React from 'react';
import ExpenseList from './expenseList';
import ExpenseListFilters from './ExpenseListFilters';
import ExpensesSummary from './ExpensesSummary';
const ExpensifyDashboard = () => (
    <div>
     <p> This is ExpensifyDashboard page.. </p>
      <ExpensesSummary/>
      <ExpenseListFilters/>
      <ExpenseList/>
    </div>
);

export default ExpensifyDashboard;