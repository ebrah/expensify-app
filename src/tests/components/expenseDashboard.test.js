import React from 'react';
import {shallow} from 'enzyme';
import  ExpensifyDashboard from '../../components/ExpensifyDashboard';


test('should render the expenses Dashboard page correctly ', () => {
    const wrapper = shallow(<ExpensifyDashboard/>);
    expect(wrapper).toMatchSnapshot() 
})
