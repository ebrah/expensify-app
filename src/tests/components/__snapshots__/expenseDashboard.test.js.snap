// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the expenses Dashboard page correctly  1`] = `
<div>
  <p>
     This is ExpensifyDashboard page.. 
  </p>
  <Connect(ExpensesSummary) />
  <Connect(ExpenseListFilters) />
  <Connect(ExpenseList) />
</div>
`;
